docs/Secure-Coding-Guide-for-Python/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.code-workspace
```

Comment:
I think this file could also be removed.

Reply:
Removing for now. Maybe we can have a more complete .gitignore for python later, for example:
https://www.toptal.com/developers/gitignore?templates=python,virtualenv,pycharm,visualstudiocode